configs/pie-components-config/tsconfig.json

```diff
@@ -8,11 +8,6 @@
     "sourceMap": true,
     "inlineSources": true,
     "outDir": "./compiled",
-    "baseUrl": ".",
```

Comment:
I removed these as they were causing issues when tests were referencing `@/foo.ts` due to relative path defined in the root config.

I'm not sure why the `paths` wasn't causing problems before, but I believe this is why we override the `baseUrl` / `rootDir` in all our configs.

Reply:
I'm unsure if I'm a fan of removing the `paths` property here. Can you elaborate on the issues found? I looked at the linked failure but I noticed that's for a rather new component and I don't think we've seen this issue anywhere else

Reply:
From the TS Documentation - https://www.typescriptlang.org/tsconfig#extends

>All relative paths found in the configuration file will be resolved relative to the configuration file they originated in.

Because of this, combined with the fact `pie-tag` didn't declare it's own `paths` property, the tests were essentially looking for the `/defs` directory from `node_modules/@justeattakeaway/pie-components-config/src`, as that's the config we're inheriting from.

Reply:
Is it only in tests that we're using `@/`? If so, I don't think we should adapt our tsconfig just for those. Especially as the docs say that [published libraries should not set up `paths`](https://www.typescriptlang.org/docs/handbook/modules/reference.html#paths-does-not-affect-emit).

Reply:
Nice find! Yeah they are. Shall I remove the aliases entirely and refactor the tests to use relative path @xander-marjoram?

Reply:
That sounds great to me, thanks!